samples/nrf9160/cloud_simple/README.rst

```diff
@@ -0,0 +1,110 @@
+.. _cloud_simple:
```

Comment:
Suggestion: With reference to this suggestion: https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1684#pullrequestreview-339859690 , I also suggest cloud_client as the sample name. 
```suggestion
.. _cloud_client:
```